bool             is_valid
float64          width
ros_leap/Vector3 direction
ros_leap/Vector3 elbow_position
ros_leap/Vector3 wrist_position